drivers/flash/flash_ospi_is25wx.c

```diff
@@ -224,7 +224,7 @@ static int flash_alif_ospi_read(const struct device *dev, off_t address, void *b
 	dev_data->trans_conf.wait_cycles = 0;
 	dev_data->trans_conf.addr_len = 0;
 
-	LOG_DBG("read address %ld length to read %d", address, length);
+	LOG_DBG("read address %u length to read %d", (uint32_t) address, length);
```

Comment:
These changes are already at different PR are we gonna need that separatelty anymore

Reply:
https://github.com/alifsemi/zephyr_alif/pull/49 This PR fix those warnings which are also in this PR.

Reply:
I think minimal change is take commit from other PR and keep warning fix at separate commit. It is not optimal have same changes at multiple PR. And it is good to keep commits simple.